docs/run-core-node/tier-1-orgs.mdx

```diff
@@ -29,7 +29,7 @@ That way, the validator community can experiment with a larger quorum, and can a
 
 ## Declare Your Node
 
-[SEP-20](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0020.md) is an open spec that explains how self-verification of validator nodes works. The steps it specifies are pretty simple: you set the homedomain of your validator’s Stellar account to your website, where you publish information about your node and your organization in a stellar.toml file.
+[SEP-20](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0020.md) is an open spec that explains how self-verification of validator nodes works. The steps it specifies are pretty simple: you set the home domain of your validator’s Stellar account to your website, where you publish information about your node and your organization in a stellar.toml file.
```

Comment:
homedomain is the name of the data entry maybe this should be `homedomain` in ` instead.